CMakeLists.txt

```diff
@@ -8,15 +8,14 @@ cmake_minimum_required(VERSION 3.15)
 cmake_policy(SET CMP0079 NEW)
 cmake_policy(SET CMP0091 NEW)
 
-set(version "1.4.0" CACHE STRING "Project version")
```

Comment:
Given that this variable is still used, I think it'd be good practice to always define it in CACHE, but default to an empty String. Furthermore I think we should update its name and description like this:
```
set(RELEASE_ID CACHE STRING "The ID of the current release (used for representation in the UI)")
```